### PR TITLE
Onboarding/Offboarding documentation

### PR DESCRIPTION
**Pull request by kfriedman**
Hello!

So, this is an attempt to address #45 - creating more thorough onboarding/offboarding documentation for developers... especially with a new developer joining our team shortly! This builds on (and consolidates) the awesome work done by @nodanaonlyzuul in https://github.com/NYPL/onboarding-offboarding.

Hopefully, we can iterate on this and create a great resource for new developers. I know this is very genera (and lacking in details) l for SimplyE but perhaps we can continue to work on that.

Please share your thoughts and any suggestions. Also, please add anyone you think would be relevant as a reviewer (or yourself!).

### CODE REVIEW

File: other/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Data Warehouse
+
+* [Data Warehouse Documentation](https://github.com/NYPL/data-warehouse)
+
+## Shared Collection Catalog/ReCAP
+
+* [Shared Collection Catalog UI](https://www.nypl.org/research/collections/shared-collection-catalog/)
+* [“Best Of” ReCAP For Developers](https://docs.google.com/document/d/1-bZsN4sRay2Vnnov-4t_D56kBnIFbyQWr8U0ZGXdUso)
+* [Request Architecture](https://docs.google.com/presentation/d/1Tmb53yOUett1TLclwkUWa-14EOG9dujAyMdLzXOdOVc)
```
Comment (rhernand3z) [inline other/README.md]: I would add the CancelRequest architecture as well.

Comment (kfriedman) [reply other/README.md]: Please feel free to add it. I think better to have more stuff than less... we can reorganize as necessary. Thanks!

File: on-off-board/onboarding.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        - [nypl-digital-dev](https://nypl-digital-dev.signin.aws.amazon.com/console)
+        - [nypl-sandbox](https://nypl-sandbox.signin.aws.amazon.com/console)
+        
+### Less commonly used accounts
+
+- NYPL Platform
+    - [https://platform.nypl.org](https://platform.nypl.org)
+- Loggly
+    - [https://nypl.loggly.com](https://nypl.loggly.com)
+- Bamboo*
```
Comment (nodanaonlyzuul) [inline on-off-board/onboarding.md]: Erroneous asterisk strikes again?

File: other/README.md (hunk 1 above)
Comment (rhernand3z) [reply other/README.md]: I added it.

I've also added a section `Web Applications` in *Other* (maybe there is a better place for this... for now I will leave as is) that lists all of the SPA's written by framework React or Angular.

It is easy to get lost in what app belongs to what repo and what is the URL. Hopefully, we can keep this list up to date when we create/deprecate applications.

@EdwinGuzman -- If I missed any App, please add or let me know.

CC: @kfriedman

Comment (EdwinGuzman) [reply other/README.md]: You could add the Header app since that's a React app.

**Pull request merged**